spi_nand_flash/examples/nand_flash/main/idf_component.yml

```
@@ -0,0 +1,4 @@
+dependencies:
+  espressif/spi_nand_flash:
+    version: '0.1.0'
```

```suggestion
    version: '*'
```

otherwise you'll have to update this line every time you make the new release